website/content/en/wgs/infra-lifecycle/_index.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Infrastructure Lifecycle Working Group
+list_pages: true
+---
+
+The [charter](./charter/charter.md) describes the goals and activites of this group.
```

Comment:
In the deploy preview I get a 404 for the charter. It looks like the `charter.md` is not needed.

This is the link that fails: https://deploy-preview-751--tag-app-delivery.netlify.app/wgs/infra-lifecycle/charter/charter.md

```suggestion
The [charter](./charter) describes the goals and activities of this group.
```

Reply:
Fixed, ty!